emission/net/ext_service/geocoder/nominatim.py

```diff
@@ -15,18 +15,18 @@
 
 try:
     googlemaps_key_file = open("conf/net/ext_service/googlemaps.json")
-    GOOGLE_MAPS_KEY = json.load(googlemaps_key_file)["api_key"]
+    GOOGLE_MAPS_KEY = json.load(googlemaps_key_file)["access_token"]
 except:
     print("google maps key not configured, falling back to nominatim")
 
 try:
     nominatim_file = open("conf/net/ext_service/nominatim.json")
-    NOMINATIM_QUERY_URL = json.load(nominatim_file)["query_url"]
+    NOMINATIM_QUERY_URL = json.load(nominatim_file)["base_url"]
```

Comment:
why not just change your conf file to use `query_url` instead of `base_url`?
In general, you want to adapt new code to what already exists instead of changing existing code and risking a regression. If you can't do that for some reason, you need to explain why